CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+## How to contribute to SST-Macro
+
+#### **Source Code Repository**
+
+* SST-Macro is the repository for the Sandia developed SST Simulation Macro.  The repository is hosted on [GitHub](https://github.com).
```

Comment:
For information on the discrete event simulation core, see [SST](...sst-core github...)